app/models/translation_attribute.rb

```diff
@@ -0,0 +1,3 @@
+class TranslationAttribute < ApplicationRecord
+  belongs_to :translation
```

Comment:
Does this need the validations (on `value` and `translation`) that the model used to have pre-rename?

Reply:
No need for translation, rails now has belongs_to as required by default, I believe. Adding one on value isn't a bad idea, though.